Update Query Performance docs (#20319)
supabase/supabase: apps/docs/pages/guides/platform/performance.mdx

Thankfully there are solutions to all these issues, which we will cover in the following sections.

<Admonition type="note">

> If you're seeing `insufficient privilege` error when viewing the Query Performance page from the dashboard. You'll need to run this command:

Reviewer: ```suggestion
If you're seeing an `insufficient privilege` error when viewing the Query Performance page from the dashboard, run this command:
```
